samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -0,0 +1,33 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+    Sample SAM Template for {{ cookiecutter.project_name }}
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+    Function:
+        Timeout: 60
+
+
+Resources:
+
+    PowerShellFunction:
+        Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+        Properties:
+            CodeUri: ./artifacts/Function.zip
+            Handler: Function::Function.Bootstrap::ExecuteFunction
+            Runtime: dotnetcore2.1
```

Comment:
Is powershell only supported in dotnetcore2.1? Do we want to hardcode that or have some logic like:

```jinja
            {%- if cookiecutter.runtime == 'dotnetcore2.0' %}
            Runtime: {{ cookiecutter.runtime }}
            {%- elif cookiecutter.runtime == 'dotnetcore2.1' or cookiecutter.runtime == 'dotnet' %}
            Runtime: dotnetcore2.1
            {%- endif %}
```

That will be parsed at runtime when the customer pass in the flag ``--runtime <value>``

Reply:
It only works on 2.1 so I hard coded that in, it won't work on 2.0